templates/start.html.tmpl

```diff
@@ -3,7 +3,7 @@
 <head>
 <meta charset="utf-8">
 <meta name="viewport" content="width=device-width, initial-scale=1">
-<link rel="stylesheet" href="/style.css">
+<link rel="stylesheet" href="style.css">
```

Comment:
We need to keep the `/` for when we have other pages that are at different depths.